WalletWasabi.Fluent/Views/Wallets/Send/PrivacyWarningsView.axaml

```diff
@@ -153,7 +157,8 @@
           <Border Classes="warning">
             <DockPanel>
               <PathIcon />
-              <TextBlock Text="{Binding CoinCount, StringFormat='Transaction consolidates over {0} coins.'}" />
+              <TextBlock Text="{Binding CoinCount, StringFormat='Transaction consolidates over {0} coins.'}" 
+                         ToolTip.Tip="A lot of coins will be used to make this transaction. This can potentially reduce the privacy of your transaction."/>
```

Comment:
The first sentence is repetitive imo.

```suggestion
                         ToolTip.Tip="This might reduce the privacy of your transaction."/>
```

Reply:
Good point. I think it is not and the word `uses` should have been used instead.